rfcs/convergence/slot-null-rendering.md

```diff
@@ -0,0 +1,261 @@
+# RFC: Slot null rendering refactoring
+
+---
+
+_@bsunderhus_
+
+## Summary
+
+`null rendering` is the process of making a `slot` render nothing. This term will be used thoroughly in this RFC.
+
+This RFC proposes a solution for a recurring problem involving slots with no `children` property into 2 steps:
+
+1. `getSlots` should stop verifying null rendering (verifying if a slot should of not be rendered) by the existence of the property `children`
+2. Since some slots have default props, verification must be done on shorthand declaration to ensure that those slots will be null rendered (or not) even presenting default props
```

Comment:
nit:
```suggestion
2. Since some slots have default props, verification must be done on shorthand declaration to ensure that those slots will be null rendered (or not) even when presented with default props.
```